code/game/machinery/deployable.dm

```diff
@@ -271,24 +274,45 @@
 	icon = 'icons/obj/dropwall.dmi'
 	icon_state = "dropwall"
 	item_state = "grenade"
-	mode = NORTH
+	mode = AUTO
 	var/uptime = DROPWALL_UPTIME SECONDS
```

Comment:
why are you multiplying this by seconds here? The MINUTES define already does this
```dm
#define MINUTES SECONDS*60
```

Reply:
Originally it was 60 seconds, forgot to change it over when someone requested it to be changed to a minute